atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -110,6 +111,7 @@ class OrchestrateBuildPlugin(BuildStepPlugin):
     """
 
     EXCLUDE_FILENAME = 'exclude-platform'
+    UNREACHABLE_CLUSTER_LOAD = -1
```

Comment:
I'm not sure about that. Using a non-numeric value internally sounds like a good idea - but what if any other plugin would want to reuse that function?

I think its worth keeping it like this - using negative values as error codes